h2o-r/tests/testdir_munging/unop/runit_mean_axis.R

```
@@ -10,7 +10,7 @@ test.mean <- function(){
   fr <- h2o.createFrame(rows = 100, cols = 10, categorical_fraction = 0,
                     factors = 0, integer_fraction = 1.0, integer_range = 100,
                     binary_fraction = 0, time_fraction = 0, string_fraction = 0,
-                    has_response = FALSE)
+                    has_response = FALSE,seed = 123456789)
```

Just for tracking: the static seed hides the failure.